presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -17,24 +17,33 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.io.OutputStream;
 import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Properties;
 
 import static com.teradata.prestomanager.agent.CommandExecutor.executeCommand;
 import static com.teradata.prestomanager.agent.PackageApiUtils.checkRpmPackage;
+import static com.teradata.prestomanager.agent.PackageApiUtils.createDir;
+import static com.teradata.prestomanager.agent.PackageApiUtils.deleteDirContents;
 import static com.teradata.prestomanager.agent.PackageApiUtils.fetchFileFromUrl;
+import static com.teradata.prestomanager.agent.PackageApiUtils.untarFile;
 import static java.lang.String.format;
-import static java.nio.file.Files.createDirectory;
 import static java.nio.file.Files.isDirectory;
+import static java.nio.file.Files.newOutputStream;
 import static java.nio.file.Files.write;
+import static java.nio.file.StandardOpenOption.CREATE_NEW;
 import static java.util.Objects.requireNonNull;
+import static java.util.UUID.randomUUID;
 
 public final class PrestoInstaller
         implements PrestoCommand
 {
     private static final Logger LOGGER = Logger.get(PrestoInstaller.class);
     private static final Path CATALOG_DIR = Paths.get("/etc/presto/catalog");
+    private static final Path TAR_INSTALLATION_DIR = Paths.get("/usr/presto");
+    private static final Path TAR_DATA_DIR = Paths.get("/usr/data");
```

Comment:
The installation directory should be relative to the user's home directory. I recommend `~/presto-manager/presto/` as the root directory for the installation, containing directories for configuration, logs, and Presto intself.

To get the home directory, use `System.getProperty("user.home")`.